build.sbt

```diff
@@ -2,9 +2,9 @@ organization := "com.metamx"
 
 name := "rainer"
 
-scalaVersion := "2.11.7"
+scalaVersion := "2.11.8"
 
-crossScalaVersions := Seq("2.10.5", "2.11.7")
+crossScalaVersions := Seq("2.10.6", "2.11.8")
```

Comment:
It might make sense to add
```
// Target Java 7
scalacOptions += "-target:jvm-1.7",
javacOptions in compile ++= Seq("-source", "1.7", "-target", "1.7"),
```
To make sure that compiled classes will be JDK7 compatible even in case of compiling with JDK8.